src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -641,20 +666,29 @@ available on the [Fuchsia devsite].
 
 ### Running the compiler test suite
 
-Pre-requisites for running the Rust test suite on Fuchsia are:
-1. Checkout of Rust source.
-1. Setup of `config-env.sh` and `config.toml` from "[Targeting Fuchsia with a compiler built from source](#targeting-fuchsia-with-a-compiler-built-from-source)".
-1. Download of the Fuchsia SDK. Minimum supported SDK version is [9.20220726.1.1](https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core/linux-amd64/+/version:9.20220726.1.1)
+To run the Rust test suite on an emulated Fuchsia device, you must install the
+Rust compiler locally. See "[Targeting Fuchsia with a compiler built from source](#targeting-fuchsia-with-a-compiler-built-from-source)"
+for the steps to build locally. By default, the Rust compiler installs itself to
+`/usr/local` on most systems. You may want to install it to another location by
+setting a custom prefix in `config.toml`:
+
+```toml
+[install]
+prefix = "<RUST_INSTALL_PREFIX>"
```

Comment:
Maybe an example of using a local `install/` directory inside the source tree would be helpful? Specifically I think suggesting something that doesn't require `sudo` is a good idea :)

It looks like we use an absolute path for this one so unfortunately it still requires the user to fill something in.

Reply:
Added.